react/package-lock.json

```diff
@@ -136,12 +136,12 @@
       }
     },
     "node_modules/@ant-design/icons": {
-      "version": "5.2.6",
-      "resolved": "https://registry.npmjs.org/@ant-design/icons/-/icons-5.2.6.tgz",
-      "integrity": "sha512-4wn0WShF43TrggskBJPRqCD0fcHbzTYjnaoskdiJrVHg86yxoZ8ZUqsXvyn4WUqehRiFKnaclOhqk9w4Ui2KVw==",
+      "version": "5.3.0",
```

Comment:
Only `package-lock.json` has been updated. If you want to upgrade `@ant-design/icons`, please make sure to update `package.json` as well.